addons/website/views/website_views.xml

```diff
@@ -314,7 +314,7 @@
                                 <div t-else="" class="o_button_area">
                                     <button type="object" name="button_choose_theme" class="btn btn-primary">Use this theme</button>
                                     <hr t-if="record.url.value"/>
-                                    <a role="button" t-if="record.url.value" class="btn btn-secondary" t-att-href="record.url.value" target="_blank">Live Preview</a>
+                                    <button role="button" type="edit" t-if="record.url.value" class="btn btn-secondary">Live Preview</button>
```

Comment:
Don't know if it is important but won't `edit` go in the form view in edit mode ? I think there is another type for readonly ?